config/environments/production.rb

```diff
@@ -102,6 +102,7 @@
     :authentication => ENV['SMTP_AUTH_METHOD'] || :plain,
     :openssl_verify_mode => ENV['SMTP_OPENSSL_VERIFY_MODE'] || 'peer',
     :enable_starttls_auto => ENV['SMTP_ENABLE_STARTTLS_AUTO'] || true,
+    :ca_file => ENV['SMTP_SSL_CERT_FILE'] || "/etc/ssl/certs/ca-certificates.crt",
```

Comment:
There are more than one way to fix this issue.
We could add the env variable in the docker file and not provide a default value in the production.rb
That way we only fix the one use case (docker containers) we have proper control over.
I can change the PR to reflect this is it is the consensus.